### DISCUSSION TITLE
Contributions wanted!

### DISCUSSION THREAD

**Discussion by nicks96432** [nicks96432/mltd-asset-downloader#23] [Announcements]
This project is bigger than I thought. I didn't expect that implementing a unity asset bundle parser will take so much time. 🤔
So, as title, please help. Any contributions is welcomed. 🤗

**Comment by 573932225** [comment]
这个项目有没有群可以讨论？

**Reply by nicks96432** [reply]
0 以後應該也不會有群
討論在這裡就好
